.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -515,7 +536,7 @@ jobs:
           git fetch https://github.com/WebAssembly/wasi-libc \
             8f5275796a82f8ecfd0833a4f3f444fa37ed4546
           git checkout FETCH_HEAD
-          make -j \
+          make \
```

Comment:
Using `make -j` can speed up the build process, did you find error when building with `-j` flag?

Reply:
I must have removed it by mistake, I put it back. The `multi-tier-jit` test is failing in the CI, it doesn't look related to this PR.